### DISCUSSION TITLE
Sonoff POW R3 shortest telemetry interval?

### DISCUSSION THREAD

**Discussion by ombonbon** [arendst/Tasmota#17358] [Q&A] [answered]
Hallo,

I just bought a Sonoff POW R3 and would like to flash tasmota to it. Before flashing, I would like to know if anyone has experience configuring the telemetry interval to a very short one. Also, what is the effect to the accuracy of measurement values?
Finally, what would be the shortest interval suggested?

Thanks (and sorry for this Noob's question)....

**Comment by barbudor** [comment, marked as answer]
Telemetry minimal interval is 10 seconds by design
To track more closely, you can use PowerDelta to send additional measuments based on variations.

I often use the standard Teleperiod of 5min along a PowerDelta of 10-15%

**Comment by ombonbon** [comment]
Thanks @barbudor for your suggestion. Do I understand correctly that using PowerDelta, the device will also send MQTT message when power is fluctuating more/less than configured PowerDelta parameter value?

**Reply by barbudor** [reply]
Indeed
You can specify the threshold either as a percentage or  an absolute value
See the docs

**Reply by sfromis** [reply]
As a starting point, I'd suggest a watt-based delta, as a percent-based one is rather finicky at low wattages. Something like `PowerDelta 110` for 10W change as the trigger, depending on what you are monitoring.

**Reply by ombonbon** [reply]
@barbudor I find only this [page](https://tasmota.github.io/docs/devices/Sonoff-Pow/#official-sources). I see also "MaxPower", "MaxPowerHold" and "MaxPowerWindow" parameters mentioned with brief description. Is there any other page (maybe you could suggest the link here) where all other paramaters are explained?
@sfromis how do we state that we want to use watt-based or percent-based delta? Is it by entering "PowerDelta 10W" or "PowerDelta 10%" in the console?

**Reply by ombonbon** [reply]
@barbudor sorry, I browse around the github page and found [this](https://tasmota.github.io/docs/Commands/#power-monitoring). Thank you so much sir.
@sfromis also thanks to you! :)
